src/openms/include/OpenMS/APPLICATIONS/OpenSwathBase.h

```diff
@@ -423,6 +422,20 @@ class TOPPOpenSwathBase :
       Param tsv_reader_param = TransitionTSVFile().getDefaults();
       OpenSwath::LightTargetedExperiment irt_transitions = loadTransitionList(tr_type, irt_tr_file, tsv_reader_param);
 
+      // If pasef flag is set, validate that IM is present
+      if (pasef)
+      {
+        auto transitions = irt_transitions.getTransitions();
```

Comment:
can this be a const auto& ? might otherwise copy transitions (I did not check the type returned by getTransitions)

Reply:
I think that it is returning an address already? https://openms.de/current_doxygen/html/structOpenSwath_1_1LightTargetedExperiment.html#a23a85c8da00a9f39f0c9ce6e15bc5a32

Not sure if I have to make it auto& to actually get it to return an address though

Reply:
Yes. Auto does never deduce references so it will make a copy of the vector.